docs/tutorials/bootloader.md

```diff
@@ -135,6 +135,70 @@ This configuration adds a new region, named header, after the bootloader and bef
 +-------------------+   BOOTLOADER_ADDR == Start of ROM
 ```
 
+
+### Prebuilt bootloaders
+
+Prebuilt bootloader mode is an option to merge prebuilt bootloaders to your application. `mbed-os\features\FEATURE_BOOTLOADER\` contains the available supported targets with the corresponding prebuilt bootloader organized in folders specific to targets. For example: `FEATURE_BOOTLOADER/targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F429xI/TARGET_NUCLEO_F429ZI/mbed-bootloader-block_device-sotp-v3_4_0.bin`.
+
+Please note that this bootloader is not the same as the public `mbed-bootloader`, which Mbed OS uses to generate the binaries in `mbed-os`.
+
+There are two ways to test prebuilt bootloader mode. 
+
+You can add the BOOTLOADER feature into `mbed_app.json` located in the root application directory:
+
+   ```
+    "target_overrides": {
+        "*": {
+            "target.features_add": ["BOOTLOADER"]
+        }
+    }
+   ```
+
+Altenratively, you can add `-C 'target.features_add=["BOOTLOADER"]'` to your `mbed compile` command-line arguments.
+
+Please see the [bootloader example](https://github.com/ARMmbed/mbed-os-example-feature-bootloader) for an example on how to use the bootloader feature.
```

Comment:
General question. Should this example be added into examples.json?

Reply:
I think should be added.
But also think we need to understand better the differences and explain multiple repositories accordingly, for example:

- mbed-os-example-feature-bootloader: example on how to include a bootloader into an application
- mbed-os-example-bootloader: simple example on how to create a basic bootloader
- mbed-bootloader: Arm reference bootloader to enable device management with Pelion

I find a few names of repositories a bit confusing. Maybe we could rename this?

mbed-os-example-feature-bootloader --> mbed-os-example-using-bootloader